### ISSUE TITLE
SALAI Token expires any ideas on how to update the token automatically?

### ISSUE THREAD

**Issue by SaschaHeyer** [erictik/midjourney-api#227]
### the code that reproduces this issue or a replay of the bug

/

### Describe the bug

The token that is generated in the browser obviously expires and leads to the following error

````
Failed to get application_commands for command settings
```` 

Does someone have an intelligent idea on how to update the token automatically?

### error log

/

**Comment by codsw** [comment]
Error: SalaiToken are required at (node_modules/midjourney/libs/midjourney.js:16:18)

**Comment by dinglewanker** [comment]
Yes.  Would be nice to automate getting the token.  I was thinking might be able to make a chrome plugin that could grab it and send it to a endpoint?

@codsw I don't think you understand his question.   He knows where to put the token.  But when it expires it gives a error.  So looking for a way to automate getting the token from a web browser.

**Comment by NatalieM11** [comment]
try this fix. it's working for me - https://github.com/erictik/midjourney-api/pull/256/files
